bundle/edu.gemini.shared.gui/src/main/scala/edu/gemini/shared/gui/textComponent/NumberField.scala

```diff
@@ -31,6 +30,9 @@ class NumberField(d: Option[Double], allowEmpty: Boolean, format: java.text.Form
 
   def valid(d:Double):Boolean = true
 
+  def value: Option[Double] = Option(peer.getValue).map(_.asInstanceOf[Double])
+  def value_=(v: Double): Unit = peer.setValue(v)
+
```

Comment:
This is to simplify setting the value on a `FormattedTextField`, since using `text_=` does not have the required effects and this class seems to be lacking in Scala Swing.